### ISSUE TITLE
Add remaining configuration to uSync json schema

### ISSUE THREAD

**Issue by Nuklon** [Jumoo/uSync.Complete.Issues#215]
**Is your feature request related to a problem? Please describe.**
Right now the appsettings-schema.usync.json file doesn't contain anything for uSync Complete, e.g., the Publisher settings are completely absent.
https://docs.jumoo.co.uk/usync/complete/setup/publisher/config
https://docs.jumoo.co.uk/usync/complete/guides/Exporter/#exporter-settings
(maybe more)

**Describe the solution you'd like**
Add these to the schema (or add a separate json file for the complete schema), so there's some code completion 😎

**Comment by KevinJump** [comment]
yeah - uSync.Complete isn't generating its schema (yet) - we will take a look.
